src/components/SpaceLayout/SpaceLayout.tsx

```diff
@@ -114,7 +114,7 @@ export function SpaceLayout(props: {
                 <SearchModal
                     spaceId={contentTarget.spaceId}
                     revisionId={contentTarget.revisionId}
-                    spaceTitle={customization.title ?? space.title}
+                    spaceTitle={space.title}
```

Comment:
I think it's a breaking change for users who have set the title in customization.

What we really need is:

```siteSpaceCustomization.title ?? space.title```

but do we have access to it?